### PR TITLE
Resolve issue #1

### LINKED ISSUE #1: Travis CI build issue
The Travis CI build will be failed when using the `composer install --prefer-lowest` command running.

After using above command, it will install `laravel/framework:6.0` version.

After tracing these failed build, the exception message is as follows:

```Bash
1) GNAHotelSolutions\LaravelTrailingSlash\Tests\TrailingSlashUrlGeneratorTest::url_ends_with_trailing_slash
BadMethodCallException: Method Illuminate\Support\Str::afterLast does not exist.
```

According to the [Laravel-News reference](https://laravel-news.com/laravel-6-5), it seems that the `Str::beforeLast` and `Str::afterLast` methods are available since `Laravel 6.5` version at least.

To resolve this issue, I think it will use `^6.5` version definition for `illuminate/support` inside `require` block on `composer.json`.

### PR DESCRIPTION
**Pull request by peter279k**
Closes #1.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+build
+composer.lock
+docs
+vendor
+coverage
+.phpunit.result.cache
```
Comment (peter279k) [inline .gitignore:6]: Add `.gitignore` file to define these files are not under Git version control.

File: .travis.yml [hunk 2]
```diff
@@ -3,6 +3,7 @@ language: php
 php:
   - 7.2
   - 7.3
+  - 7.4
```
Comment (peter279k) [inline .travis.yml:6]: Add `php-7.4` version during Travis CI build.

File: composer.json [hunk 3]
```diff
@@ -22,7 +22,7 @@
     ],
     "require": {
         "php": "^7.2",
-        "illuminate/support": "^6.0|^7.0"
+        "illuminate/support": "^6.5|^7.0"
```
Comment (peter279k) [inline composer.json:25]: It's related to issue #1.

File: phpunit.xml.dist [hunk 4]
```diff
@@ -22,7 +22,7 @@
     <logging>
         <log type="tap" target="build/report.tap"/>
         <log type="junit" target="build/report.junit.xml"/>
-        <log type="coverage-html" target="build/coverage" charset="UTF-8" yui="true" highlight="true"/>
```
Comment (peter279k) [inline phpunit.xml.dist:25]: These PHPUnit attributes are not available on `PHPUnit 8.x` version. And they should be removed.

The related warning message is as follows:

```
Warning - The configuration file did not pass validation!
  The following problems have been detected:

  Line 25:
  - Element 'log', attribute 'charset': The attribute 'charset' is not allowed.
  - Element 'log', attribute 'yui': The attribute 'yui' is not allowed.
  - Element 'log', attribute 'highlight': The attribute 'highlight' is not allowed.

  Test results may not be as expected.
```

File: composer.json [hunk 5]
```diff
@@ -22,10 +22,10 @@
...
     },
     "require-dev": {
-        "orchestra/testbench": "^4.0",
+        "orchestra/testbench": "^4.0|^5.0",
```
Comment (peter279k) [inline composer.json:28]: Defining `^5.0` for `Laravel 7.x` version.

**Comment by Lloople** [comment]
Thank you! 🥳

**Pull request merged**